scripts/python/tao_build.py

```diff
@@ -227,6 +227,45 @@ def configure_compiler(root, args):
                 f.write("test --remote_cache={}\n".format(token))
     logger.info("Stage [configure] success.")
 
+def mkldnn_build_dir(root=None):
+    if root is None:
+        root = get_source_root_dir()
+    return os.path.join(root, "tao", "third_party", "mkldnn", "build")
+
+def mkl_install_dir(root):
+    return os.path.join(mkldnn_build_dir(root), "intel")
+
+def config_mkldnn(root, args):
+    build_dir = mkldnn_build_dir(root)
+    ensure_empty_dir(build_dir, clear_hidden=False)
+    mkl_dir = mkl_install_dir(root)
+    ensure_empty_dir(mkl_dir, clear_hidden=False)
+    # download mkl-lib/include
+    with cwd(mkl_dir):
+        download_cmd = '''
+          curl -fsSL https://hlomodule.oss-cn-zhangjiakou.aliyuncs.com/mkl_package/mkl-static-2022.0.1-intel_117.tar.bz2  | tar xjv
+          curl -fsSL https://hlomodule.oss-cn-zhangjiakou.aliyuncs.com/mkl_package/mkl-include-2022.0.1-h8d4b97c_803.tar.bz2 | tar xjv
+        '''
```

Comment:
This part of code is used for tao TF bridge, which uses CMake as build tool instead of bazel. In the long term, we should use bazel for tao tf bridge  as well and then it would be better to use the oneDNN in tf_community.